edx_repo_tools/release/tag_release.py

```diff
@@ -0,0 +1,633 @@
+#!/usr/bin/env python
```

Comment:
Because this moved the entire file, and made changes, it's hard to know what you've done.